decadog/src/command/sprint.rs

```diff
@@ -260,6 +281,25 @@ fn finish_sprint(settings: &Settings) -> Result<(), Error> {
     }
 
     println!();
+    // Update title with number of planned and completed points this sprint
+    // Prompt user for number of planned points in the sprint
+    let planned_points_str = Input::<String>::new()
+        .with_prompt("Points planned this sprint (q: quit)")
+        .interact()?;
+    if planned_points_str == "q" {
+        return Ok(());
+    }
+    let planned_points: u32 = planned_points_str.parse().map_err(|_| Error::User {
+        description: format!("Invalid number of planned points {}.", &planned_points_str),
+    })?;
+    // New title: Sprint <milestone_number> [<points done in sprint>/<points planned> + <points
+    // done out of sprint>]
+    let new_title = format!(
+        "Sprint {} [{}/{} + {}]",
+        open_milestone, points_done_in_sprint, planned_points, points_done_out_of_sprint
```

Comment:
`open_milestone` (milestone number) is not the same as sprint number - you could either prompt for the user to enter the sprint number, or just do something like `format!("{} [...]", milestone.title)` to reuse the existing title

Reply:
Good catch!